enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+
+```
+$ alpha audit --help
+
+Flags:
+  -h, --help                     help for audit
+  -i, --index-image              index image to get package from. 
+  -c, --container-tools          the container tool used to login in the registry and pull and extract the images. One of: [docker, podman] (default "docker")
+  -w, --wait-time duration       minutes to wait for the report to be complete. Example: 60m (default timeout is 30m)       
+  -o, --output                   output format for results. Valid values: xlsx, json (default "xlsx")
+  -s, --scorecard bool           set false to disable the scorecard checks. (default true). Note that scorecard test requires a cluster running. 
+  -v, --validation bool          set false to disable the bundle validation checks. (default true). 
+  -d, --data                     set false to disable the data report. It is useful for who would like to audit the index via only run the bundle and scorecard checks. (default true).
+  -h, --head-only                set the flag to audit only the operator bundles which are the head of the channel. (default all bundles of the index catalog image will be used)
+  -w, --warning                  set false to disable the report of warning(s) found in the checks (default warning and errors will be reported)
+```
+
+Note that the command will:
+- Use the index image to gathering the data
+- Download the operator bundles
+- Use SDK commands to run the checks 
+- Output the results
+
+**NOTE** If we move forward with this command as part of opm it would be like: `opm alpha index audit --help`
+ 
+### User Stories 
+ 
+- As an admin and or maintainer of OLM index catalog, I would like to check if all bundle operators in the catalogue are respecting all expected criteria so that, I can ensure a better quality of all bundles in the catalogue.
+- As an admin and or maintainer of OLM index catalog, I would like to easily know more about the bundles in the catalogue so that, I can easily provide strategies which will be useful for the biggest part of the authors of the projects published on it.
+- As an admin and or maintainer of OLM index catalog, identify problems and concerns in the operators bundles of some catalogue so that, I can easily decide if they should or not be removed from the index catalog.
+- As an admin and or maintainer of OLM index catalog I would like to audit the OLM index catalog by running custom tests for each operator bundle that, I can use it to verify if all operator bundles will work as expected.
+- **(Optional)** As operator developer I would like to configure scorecard via SDK in a bundle of a project which is a NO-SDK layout so that, I could check my bundle with scorecard feature. For more info check [Scorecard config manifests](#scorecard-config-manifests). We can address it in the audit command as well at the first moment with some implications. Then, it is not required at all if the solution get shipped on SDK too.
```

Comment:
Running scorecard against a non-SDK project seems out of context compared to the rest of this EP.  Maybe I'm missing something.

Reply:
In the catalog we have the bundles, I just want a way to scaffold the 1 file that setups the scorecard for the bundle when it does not exist. why not audit does that? Because it can change then I will run the sdk scorecard with SDK bin version A and the config is not compatible.

Reply:
I removed it and clarified. Also, the EP was improved to cover all concerns about this subject. Please, feel free to check and let me know what do you think.